packages/design-system/package.json

```diff
@@ -11,19 +11,19 @@
     "url": "https://github.com/livechat/design-system"
   },
   "typings": "src/interfaces/index.d.ts",
-  "main": "dist/design-system.cjs.js",
-  "module": "dist/design-system.es.js",
-  "jsnext:main": "dist/design-system.es.js",
-  "style": "dist/design-system.css",
+  "main": "../../dist/core/design-system.cjs.js",
```

Comment:
woaaah, are you sure? shouldn't we change the path on bamboo CDN? `design-system.cjs.js` should stay in the package directory imho

Reply:
Well, with monorepo like this one created with `lerna` there should be single folder for artifacts.

Reply:
but this `main` property is package only, monorepo thing doesn't matter here

Reply:
https://take.ms/Z0ynv - take a look at this. This is a published design-system library in AA. If you change main etc. it will break the package probably

Reply:
good catch - fixed in a0846f4